mappings/net/minecraft/client/render/BufferRenderer.mapping

```diff
@@ -1,2 +1,16 @@
 CLASS net/minecraft/class_286 net/minecraft/client/render/BufferRenderer
+	FIELD field_38982 currentVertexBuffer Lnet/minecraft/class_291;
 	METHOD method_34420 unbindAll ()V
+	METHOD method_43433 draw (Lnet/minecraft/class_287;)V
+		ARG 0 builder
+	METHOD method_43434 bindAndSet (Lnet/minecraft/class_291;)V
+		ARG 0 vertexBuffer
+	METHOD method_43435 bindAndSet (Lnet/minecraft/class_293;)Lnet/minecraft/class_291;
+		ARG 0 vertexFormat
+	METHOD method_43436 resetCurrentVertexBuffer ()V
+	METHOD method_43437 postDraw (Lnet/minecraft/class_287;)V
```

Comment:
It was named `postDraw` previously. Re-checking this, it could be a yarnbug

Reply:
I am looking at this method now while updating one of my mods.

I agree that it's not postDraw, enbrain is correct I think. IDK what a good name for it would be since I'm not a renderer, but drawElements maybe since that's the G11 call it uniquely leads to.

Reply:
BufferRenderer methods renamed to drawWithShader/drawWithoutShader